CHANGELOG.md

```diff
@@ -1,3 +1,29 @@
+### citus v10.0.2 (March 3, 2021) ###
+
+* Fixes insert query with CTEs/sublinks/subqueries etc
+
+* Removes limits around long table names
+
+* Uses `PROCESS_UTILITY_QUERY` in utility calls
+
+* Preserves colocation with procedures in `alter_distributed_table`
+
+* Grants `SELECT` permission on `citus_tables` view to `public`
+
+* Fixes a bug with `UNION ALL` on PG 13
+
+* Grants read access for columnar metadata tables to unprivileged user
```

Comment:
Maybe we should rewrite this as:
```suggestion
* Grants `SELECT` permission on columnar metadata tables to `public`
```

As you wrote for `citus_tables`:
>Grants `SELECT` permission on `citus_tables` view to `public`

----

Plus, I think we should group those two entries